src/os/ui/menu/listmenu.js

```diff
@@ -127,18 +127,18 @@ os.ui.menu.list.handleListEvent = function(event) {
     var eventType = event.type.replace(os.ui.menu.list.PREFIX_REGEXP, '');
     switch (eventType) {
       case os.action.EventType.SELECT:
-        // omit selection events from the stack to reduce clutter
-        new os.command.SelectAll(source.getId()).execute();
+        // don't create and execute a command if it's simple and we don't want it on the stack
```

Comment:
Minor nit - could you also remove the `goog.require` statements for these? The ES6 transform should remove them automatically if they're not used (once we run it), but we should try to clean those up as file dependencies change.

Reply:
Thanks for the cleanup!

Reply:
Good to clean this up while we're in here 👍